spec/class.dd

```diff
@@ -100,6 +100,23 @@ $(H2 $(LNAME2 access_control, Access Control))
         The default visibility attribute is $(D public).
         )
 
+$(H2 $(LNAME2 super_class, Super Class))
+
+        $(P
+        All classes inherit from a super class. If one is not specified,
+        a class inherits from $(REF1 Object, object). `Object` forms the root
+        of the D class inheritance hierarchy.
+        )
+
+---
+class A { }     // A inherits from Object
+class B : A { } // B inherits from A
+---
+
+        $(P Multiple class inheritance is not supported.
+        However, a class can inherit from multiple
+        $(DDLINK spec/interface, Interfaces, interfaces) in addition to its super class.)
```

Comment:
might be worth noting that the parent class must be first, then interfaes listed after, separated by commas

the grammar says that too but in more grammar-ese

Reply:
Thanks, updated.